.github/workflows/ci.yml

```diff
@@ -52,6 +52,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2.5.2
```

Comment:
Why do we prefer the GitHub action over the Makefile command?
Contra:
- You would not assume that the CI does something else than you do.
- Information duplication. The `timeout` flag in the Makefile was probably intended for execution in the CI, but was not used there.

Reply:
Good question, I don't know. A potential advantage could be to use an action with a dedicated formatter that annotates results in Pull Requests. As we don't use this feature so far, I am also fine with another mechanism.

Reply:
https://github.com/openHPI/poseidon/issues/111